util/flags.py

```diff
@@ -16,8 +16,6 @@ def create_flags():
     tf.app.flags.DEFINE_boolean ('fulltrace',        False,       'if full trace debug info should be generated during training')
 
     tf.app.flags.DEFINE_string  ('train_cached_features_path',      '',          'comma separated list of files specifying the dataset used for training. multiple files will get merged')
```

Comment:
If we really drop caching for `dev` and `test`, we should name this `cached_features` or `feature_cache`.